src/qml/components/PeersIndicator.qml

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// The PeersIndicator component.
+// See reference design: https://bitcoindesign.github.io/Bitcoin-Core-App/assets/images/block-clock-connection-states-big.png
+
+import QtQuick 2.15
+import "../controls"
+
+Item {
+    id: root
+    property int numOutboundPeers: 0
+    property int maxNumOutboundPeers: 10
```

Comment:
How about making it a readonly property? Should we permit other files to edit this property outside of this file?

```suggestion
    readonly property int maxNumOutboundPeers: 10
```

Reply:
this can change when there's configuration and options handling, it's set to this value by default